core/base/planarGraphLayout/FTMTreePPCompute.h

```diff
@@ -0,0 +1,248 @@
+/// \ingroup base
+/// \class ttk::FTMTreePPCompute
+/// \author Charles Gueunet <charles.gueunet@lip6.fr>
```

Comment:
are these functions taken from charles's code?
are there code duplicates?

Reply:
Indeed, it was a quick fix in my local version, I needed to modify this class in order to be used in my code, we need a better solution indeed!